server/config.js

```diff
@@ -1,6 +1,7 @@
 const isProduction = process.env.NODE_ENV === 'production';
 const workflowAuth = (process.env.WORKFLOW_BASIC_AUTH || 'UNSET:UNSET').split(':');
 const isNotProd = process.env.IS_NOTPROD === '1';
+const showStackTraceInErrorPage = process.env.SHOW_STACKTRACE_ON_ERROR_PAGE === '1';
```

Comment:
Default this to the value of `isNotProd`.

Reply:
It defaults to false if it's not there anyway? `isNotProd` is true for demo environments that contradicts what is in the AC of the ticket.